popsborder/consignments.py

```diff
@@ -317,6 +317,61 @@ def generate_consignment(self):
         )
 
 
+class VariablePackagingConsignmnentGenerator:
+    """Generate a consignments based on existing shipment records where items per
+    box is specified per record.
+    """
+
+    def __init__(self, filename, separator=","):
+        self.infile = open(filename)
+        self.reader = csv.DictReader(self.infile, delimiter=separator)
+
+    def generate_consignment(self):
+        """Generate a new consignment"""
+        try:
+            record = next(self.reader)
+        except StopIteration:
+            raise RuntimeError(
+                "More consignments requested than number of records in shipment data"
+            ) from None
+        items_per_box = int(record["ITEMS_PER_PACKAGE"])
+        unit = record["QUANTITY_UNIT"]
+
+        # Generate items based on quantity in records.
+        # Quantity unit must be stems/items.
+        if unit in ["Stems", "Items"]:
+            num_items = int(record["QUANTITY"])
+        else:
+            raise RuntimeError(f"Unsupported quantity unit: {unit}")
```

Comment:
Can you please describe here, in the class docstring, or the PR description on the difference between this class and AQIM?

I can see the two classes are similar and we can resolve some de-duplication some other time, but it would be good to have clear specification of what are the differences.